hackathon_site/event/api_urls.py

```diff
@@ -14,6 +14,11 @@
     ),
     path("teams/leave_team/", api_views.LeaveTeamView.as_view(), name="leave-team"),
     path("teams/", views.TeamListView.as_view(), name="team-list"),
+    path(
+        "teams/team/incidents",
```

Comment:
```suggestion
        "teams/team/incidents/",
```

The trailing slash strikes again!